docs/import/manage_disaster.js

```diff
@@ -675,41 +686,85 @@ function toggleState(known) {
   }
 }
 
+const scoreAssetTypes = [
+  ['poverty', ['snap_data', 'paths'], 'Poverty'],
+  ['income', ['income_asset_paths'], 'Income'],
+  ['svi', ['svi_asset_paths'], 'SVI'],
+];
+Object.freeze(scoreAssetTypes);
+
+const assetSelectionRowPrefix = 'asset-selection-row-';
+
+/**
+ * Initializes score selector table based on {@link scoreAssetTypes} data. Done
+ * as soon as page is ready.
+ */
+function setUpScoreSelectorTable() {
+  const tbody = $('#asset-selection-table-body');
+  for (const scoreAssetType of scoreAssetTypes) {
+    const row = $(document.createElement('tr'));
+    row.append(createTd().text(scoreAssetType[2]));
+    row.prop('id', assetSelectionRowPrefix + scoreAssetType[0]);
+    tbody.append(row);
+  }
+}
+
 /**
  * Initializes the select interface for score assets.
  * @param {Array<string>} states array of state (abbreviations)
  */
 function initializeScoreSelectors(states) {
   const headerRow = $('#score-asset-header-row');
-  const tableBody = $('#asset-selection-table-body');
-  tableBody.empty();
-  headerRow.empty();
 
   // Initialize headers.
-  headerRow.append(createTd().html('Score Assets'));
+  removeAllButFirstFromRow(headerRow);
```

Comment:
I don't think this line is working, not sure why.

Reply:
Sorry, fixed in the sequel but didn't back port.